FLT/Mathlib/MeasureTheory/Measure/Comap.lean

```diff
@@ -19,17 +19,4 @@ lemma nullMeasurableSet_smul_iff_nullMeasurableSet {X : Type*} [MeasurableSpace
   apply nullMeasurableSet_smul_of_nullMeasurableSet c⁻¹ at h
   rwa [smul_smul, inv_mul_cancel₀ hc.ne', one_smul] at h
 
-lemma comap_smul {X Y : Type*} [MeasurableSpace X] [MeasurableSpace Y] {f : X → Y} {c : ℝ≥0}
```

Comment:
You can also delete the two lemmas above this one about NullMeasurableSet, they were also added to Mathlib (under slightly different names).

Reply:
Ok, thanks!

Reply:
Removed the entire file. Thanks!